temporian/implementation/numpy/data/event.py

```diff
@@ -62,6 +93,27 @@ def schema(self) -> Event:
     def __repr__(self) -> str:
         return self.data.__repr__()
 
+    def __eq__(self, __o: object) -> bool:
+        if not isinstance(__o, NumpyEvent):
+            return False
+
+        # Check equal sampling and index values
+        if self.sampling != __o.sampling:
+            return False
+
+        # check same features
+        if self.feature_names != __o.feature_names:
+            return False
+
+        # Check each feature is equal in each index
+        for index in self.data.keys():
+            for feature in self.data[index]:
+                if feature not in __o.data[index]:
+                    print(f"Feature {feature} not in {__o.data[index]}")
+                    return False
```

Comment:
Here we are checking that all of `self`'s features are in `__o`, but not other way round?

If we want to make equal events necessarily have features in the same order (which I think could make sense), we could just check that `all(f1 == f2 for f1, f2 in zip(self.data[index], __o.data[index]))` for example, which is also faster (with the `in` operator we are getting O(n^2) since we are traversing the second list for each element in the first one).

Also, would remove this print.

Reply:
Practically, features in a different event should still be the same event because they have the same information. However, I agree that is easier and faster to just check that every feature is in the same order